### DISCUSSION TITLE
Need to integrate Quickcart extension in breeze theme provide my other vendor

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#189] [1. Help]
Hello @vovayatsyuk 

I need to integrate one weltpixel quickcart extension into breeze theme.

Is it possible that it will be compatible with breeze theme for magento 2. Can you please help me out.

Thanks for any help in advance....!!!

**Comment by dimdimich** [comment]
Can you please let us know the difference between the Weltpixel Quickcart extension and https://swissuplabs.com/ajax-pro-extension-for-magento-2.html? It might be easier to use a module that is already integrated.

Also, for Weltpixel, I would suggest trying to follow the same plan that is described at https://swissuplabs.com/blog/interested-in-amasty-modules-integration-with-breeze-check-this-information/.

Please try to contact them and ask for integration.
